salalib/pointdata.cpp

```diff
@@ -3609,6 +3609,11 @@ void PointMap::mergeFromShapeMap(const ShapeMap& shapemap)
    }
 }
 
+bool PointMap::isPixelMerged(PixelRef a)

```

Comment:
Argument should be const PixelRef &a - otherwise you take an unneeded copy (and no one really knows what some of these classes do when you copy them)
Probably worthwhile to get into the habit to pass everything (except builtins) by const & unless something else is required...